File: Java/Day2/Task1.java
import java.util.Scanner;
public class Task1{
public static void main(String[] args){
	Scanner input = new Scanner(System.in);
		System.out.print("Current father's age: ");
		int fatherAge = input.nextInt();

		System.out.print("Current son's age: ");
		int sonAge = input.nextInt();

		int result = checkAge(fatherAge, sonAge);
		System.out.println("Father age will be twice of son age in " + result + " years");;
	}

	public static int checkAge(int fatherAge, int sonAge){
		int yearCount = 0;
		for (int age = 0; age < fatherAge; age++){
			if (fatherAge == age * 2){
				yearCount += age;
			}
		}
		return yearCount;
		
	}

}

File: Python/Day2/task_1.py
def get_father_and_son_age(father_age, son_age):
	father_age = (input("Enter father's age: "))
	son_age = (input("Enter son's age: "))
	year_count = 0
	for age in father_age:
		if father_age == age * 2:
			year_count += age
	return year_count

result = get_father_and_son_age(father_age, son_age)

print(f"Father age will be twice of son age in {year_count} years")
	
	